lib/ember-test-helpers/test-module.js

```diff
@@ -269,6 +271,36 @@ export default Klass.extend({
     };
   },
 
+  /*
+    Builds a version of the passed in context that contains deprecation warnings
+    for accessing properties that exist on the module.
+  */
+  _buildDeprecatedContext: function(module, context) {
+    var deprecatedContext = Object.create(context);
```

Comment:
I would prefer to avoid the `Object.create(context)` call when there are no `keysForDeprecation`, can you tweak this to be behind a guard?

Something like this slight reordering:

```
var keysForDeprecation = Object.keys(module);

if (keysForDeprecation.length === 0) {
  return this;
}

var deprecatedContext = Object.create(context);
// .... other stuff
```